README.md

```diff
@@ -1,2 +1,25 @@
 # Collabo Web App
-Content coming soon
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app) [![GitHub issues](https://img.shields.io/github/issues/code-collabo/collabo-web-app?color=red)](https://github.com/code-collabo/collabo-web-app/issues) [![GitHub pull requests](https://img.shields.io/github/issues-pr/code-collabo/collabo-web-app?color=goldenrod)](https://github.com/code-collabo/collabo-web-app/pulls)
+
+The Collabo Web App is an internal project aimed at promoting the Code Collabo Community, its projects, and programs. It also serves as a showcase for our web development tools like node-mongo and Collabo CSS framework. Opportunities for UI/UX and graphic design contributions are also available.
+
+#
+
+Find out how to use and contribute to this web application project in the [Collabo Web App Contributing Doc](https://code-collabo.gitbook.io/collabo-contributor/collabo-web-app-project/contribute-to-collabo-web-app)
+
+#
+
+To learn more about Code Collabo's community and other projects, feel free to visit the official [Code Collabo Documentation](https://code-collabo.gitbook.io/docs)
+
+## Contributors
+
+Thanks to these amazing contributors to the collabo web application project. Contributions of any kind welcome!
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<a href="https://github.com/code-collabo/collabo-web-app/graphs/contributors"><img src="https://contrib.rocks/image?repo=code-collabo/collabo-web-app" alt="All the amazing contributors of the Collabo Web App"></a>
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```

Comment:
**Review comment 4:** We have not set up @all-contributors bot for this repo, so you can remove the entire contributors section for now. This part is addressed in another issue ticket.